client/lib/perfmon/index.js

```diff
@@ -173,7 +173,7 @@ module.exports = function() {
 	if ( !initialized ) {
 		var MutationObserver = window.MutationObserver || window.WebKitMutationObserver;
 
-		if ( MutationObserver ) {
+		if ( MutationObserver && window.performance ) {
```

Comment:
It's just of a "just in case" check, and it only gets executed once so I figured "why not".